src/colorschemes.jl

```diff
@@ -111,22 +130,32 @@ struct CategoricalColorGradient <: ColorGradient
     end
 end
 
-plot_color(cg::CategoricalColorGradient, α::Number) = CategoricalColorGradient(plot_color(cg.colors, α), cg.values)
+plot_color(cg::CategoricalColorGradient, α::Number) =
+    CategoricalColorGradient(plot_color(cg.colors, α), cg.values)
 
 Base.show(io::IO, m::MIME"image/svg+xml", cg::CategoricalColorGradient) =
     show(io, m, cg[get_range(100)])
+
 function Base.get(cg::CategoricalColorGradient, x::AbstractFloat, rangescale = (0.0, 1.0))
     isfinite(x) || return invisible()
     rangescale = get_rangescale(rangescale)
     x = clamp.(x, rangescale...)
     if rangescale != (0.0, 1.0)
         x = ColorSchemes.remap(x, rangescale..., 0, 1)
     end
-    return cg.colors[x == 0 ? 1 : findlast(<(x), cg.values)]
+    return cg.colors[x == 0 ? 1 : findlast(z -> z < x, cg.values)]
```

Comment:
just curious - why this change?

Reply:
Because the former version does not work on julia 1.0

Reply:
Did you fix the RecipesPipeline bug?

Reply:
Yes, it was due to outdated precompile statements in Plots.